code/modules/mining/mine_turfs.dm

```diff
@@ -147,18 +100,16 @@ var/global/list/rockTurfEdgeCache
 
 /turf/simulated/mineral/iron
 	name = "iron deposit"
-	icon_state = "rock_Iron"
 	mineralType = /obj/item/weapon/ore/iron
 	spreadChance = 20
 	spread = 1
-	hidden = 0
+	scan_state = "rock_Iron"
```

Comment:
I've seen drones often go looking for iron, and gulag prisoners who don't know how to mine sand tend to beeline to iron as well.